src/main/java/TaskType.java

```diff
@@ -0,0 +1,3 @@
+public enum TaskType {
+    TODO, DEADLINE, EVENT
```

Comment:
I love this idea of using enums here! I didn't think of using it to differentiate task type in my code